src/main/java/com/kodilla/ecommercee/controller/group/controller/GroupController.java

```diff
@@ -0,0 +1,33 @@
+package com.kodilla.ecommercee.controller.group.controller;
+
+import com.kodilla.ecommercee.domain.Product.ProductDto;
+import com.kodilla.ecommercee.domain.groups.GroupDto;
+import org.springframework.web.bind.annotation.*;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/group/")
+public class GroupController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getGroups")
+    public List<GroupDto> getGroups() {
+        List<GroupDto> groupDtos = new ArrayList<>();
+        groupDtos.add(new GroupDto(2L,"group name 2"));
```

Comment:
teraz to mniej ważne, bo tutaj testujemy działanie kontrolera ale to powinna być lista obiektów z encji, a nie dto, bo przechowujesz je w bazie danych i gdy przekazujesz przez kontroler to za pomocą mappera powinienieś zamieniać je na dto.